mergecounts.wdl

```diff
@@ -1,17 +1,33 @@
 task MergeCounts {
+    String? preCommand
+
     Array[File] inputFiles
     String outputFile
     String idVar
     String measurementVar
-    File script
 
-    command {
-        Rscript ${script} \
-        ${idVar} \
-        ${measurementVar} \
-        ${sep=" " inputFiles} \
-        > ${outputFile}
-    }
+    # Based on a script by Szymon Kielbasa/Ioannis Moustakas
+    command <<<
+        set -e -o pipefail
+        ${preCommand}
+        R --no-save --slave <<CODE > ${outputFile}
```

Comment:
I think the `> {outputFile}` belongs after the second `CODE`. Never tested this myself but it looks like this part becomes part of the Rscript

Reply:
I know it's rather counter-intuitive, but I tested this. I initially put the `> ${outputFile}` behind the second `CODE` as well, but than the `CODE > ${outputFile}` became part of the R script (the CODE didn't terminate the script anymore).